storage/list.go

```diff
@@ -0,0 +1,79 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"github.com/fewsats/fewsatscli/client"
+	"github.com/urfave/cli/v2"
+)
+
+type File struct {
```

Comment:
we will soon switch to generated clients so we do not have to duplicate these types 🚀